packages/x-date-pickers-pro/src/SingleInputTimeRangeField/SingleInputTimeRangeField.interfaces.ts

```diff
@@ -0,0 +1,15 @@
+import { TextFieldProps } from '@mui/material/TextField';
+import { UseTimeRangeFieldProps } from '../internal/models';
+
+export interface UseSingleInputTimeRangeFieldProps<TDate> extends UseTimeRangeFieldProps<TDate> {}
+
+export type UseSingleInputTimeRangeFieldComponentProps<TDate, ChildProps extends {}> = Omit<
+  ChildProps,
+  'value' | 'defaultValue' | 'onChange' | 'onError'
+> &
+  UseSingleInputTimeRangeFieldProps<TDate>;
+
+export type SingleInputTimeRangeFieldProps<TDate> = UseSingleInputTimeRangeFieldComponentProps<
```

Comment:
Could you have a look and confirm my assumption that these `.interfaces` files only duplicate what is already in `.types` and nothing from these is used anywhere..? 🤔 
This and another `interfaces` file also references `TextFieldProps` instead of `FieldsTextFieldProps`, but that doesn't matter if these should simply be removed...

@flaviendelangle Maybe you remember why this could have happened? Did we at some point rename the files from `interfaces` to `types`? 🤔

Reply:
I think I dropped the `.interface` file of `SingleInputDateRangeField` recently.
Probably an error I did at some point with a Git merge.

Reply:
Effectively, it's not even exported from the index.ts

Reply:
`SingleInputDateTimeRangeField.interfaces.ts` does not seem to be used as well.